### DISCUSSION TITLE
:CopilotChatReview creates lsp warnings that persist after implemeting copilot's suggestions

### DISCUSSION THREAD

**Discussion by djfinnoy** [CopilotC-Nvim/CopilotChat.nvim#284] [Q&A]
I tried using `:CopilotChatReview`. It worked great, but then I noticed that copilot's feedback gets picked up by my lsp (gopls).

After refactoring my code as per copilot's suggestions, the warning persists. I could find no other way to get rid of these warnings than closing and re-opening neovim.

I would like to disable this lsp integration as I think the chat window is sufficient, but I can't see any reference on how to do this in the docs.

Is it possible to use `:CopilotChatReview` without affecting the language server?

**Comment by deathbeam** [comment]
You can set callback on the prompt to empty function and that would solve it, but ideally the workflow should be to rerun :copilotchatreview after you are done with the refactors etc. Example on how to disable the callback:

```lua
  prompts = {
    Review = {
      callback = function() end
    }
  }
```
